modules/Contacts/JAccountMissingException.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+
+class JAccountMissingException extends Exception {}
```

Comment:
Please Don't defined empty exceptions, conside using the constructor to define a default message and error code.